.github/workflows/label.yml

```diff
@@ -0,0 +1,50 @@
+name: Labeler
+on:
+  pull_request_target:
+    types: closed
+
+jobs:
+  label:
+
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      issues: write
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          sparse-checkout: .github/scripts
+      - name: Add release labels on merge
+        run: |
+          PR_NUMBER="${{ github.event.pull_request.number }}"
```

Comment:
This could also be provided through the `env` property. But this works as well.

Reply:
Don't think it is available as an env, see https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables